### PR TITLE
Feature/oe 5522

### PR DESCRIPTION
**Pull request by vetusko**
Test controller for the AdminListAutocomplete features

### CODE REVIEW

File: controllers/AdminController.php [hunk 1]
```diff
@@ -82,20 +82,23 @@ public function actionCommonDrugsDelete($ssdId)
 	 * @description Adds new drug into the site_subspecialty_drug table - AJAX call only
 	 * @param $drugId
 	 * @param $siteId
-	 * @param $subspecialty_id
+	 * @param $subspecialtyId
 	 * @return string
 	 */
-	public function actionCommonDrugsAdd($drugId, $siteId, $subspecId)
```
Comment (petergallagher) [inline controllers/AdminController.php:88]: This PHPDoc is now wrong.

File: controllers/AdminController.php [hunk 2]
```diff
@@ -59,7 +59,7 @@ public function actionCommonDrugs()
 	 * @param $ssd_id
 	 * @return html
 	 */
-	public function actionCommonDrugsDelete($ssdId)
+	public function actionCommonDrugsDelete($itemId)
```
Comment (petergallagher) [inline controllers/AdminController.php:62]: This has been changed to itemId by the javascript is still sending ssdId so delete is not working.

File: controllers/AdminController.php (hunk 1 above)
Comment (vetusko) [inline controllers/AdminController.php:88]: changed

File: controllers/AdminController.php (hunk 2 above)
Comment (vetusko) [inline controllers/AdminController.php:62]: actually the listAutocomplete.js sending itemId, and the old version of the list need to be removed

**Pull request merged**